lib/ex_plasma/output/position.ex

```diff
@@ -0,0 +1,120 @@
+defmodule ExPlasma.Output.Position do
+  @moduledoc """
+  Generates an Output position if given the:
+
+  `blknum` - The block number for this output
+  `txindex` - The index of the Transaction in the block.
+  `oindex` - The index of the Output in the Transaction.
+  """
+
+  @behaviour ExPlasma.Output
+
+  alias ExPlasma.Output
+
+  @type position() :: pos_integer()
+
+  @type t() :: %{
+          position: position(),
+          blknum: non_neg_integer(),
+          txindex: non_neg_integer(),
+          oindex: non_neg_integer()
+        }
+
+  @type validation_responses() ::
+          {:ok, t() | Output.t()}
+          | {:error, {:blknum, :cannot_be_nil}}
+          | {:error, {:blknum, :exceeds_maximum_value}}
+          | {:error, {:oindex, :cannot_be_nil}}
+          | {:error, {:txindex, :cannot_be_nil}}
+          | {:error, {:txindex, :exceeds_maximum_value}}
+
+  # Contract settings
+  # These are being hard-coded from the same values on the contracts.
+  # See: https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/contracts/src/utils/PosLib.sol#L16-L23
+  @block_offset 1_000_000_000
```

Comment:
maybe this is something that should be pulled from the contracts?

Reply:
But, `ex_plasma` doesn't pull from the contracts anymore. We'll need this setting in the library so we can build the output position.

Reply:
yes this makes sense. it would also make sense to prevent starting the ex_plasma application if this configuration is not set. but this should come from the consumer of this dependency, (for example - childchain).